dashboard/config/levels/custom/music/musiclab_intro_change_sound.level

```diff
@@ -0,0 +1,72 @@
+<Music>
+  <config><![CDATA[{
+  "published": true,
+  "game_id": 70,
+  "created_at": "2024-01-09T00:48:10.000Z",
+  "level_num": "custom",
+  "user_id": 976,
+  "properties": {
+    "encrypted": "false",
+    "background": "music-black",
+    "level_data": {
+      "library": "will",
+      "startSources": {
+        "blocks": {
+          "languageVersion": 0,
+          "blocks": [
+            {
+              "type": "when_run_simple2",
+              "id": "when-run-block",
+              "x": 30,
+              "y": 30,
+              "movable": "false",
+              "deletable": "false",
+              "next": {
+                "block": {
+                  "type": "play_sound_at_current_location_simple2",
+                  "id": "play-sound-block",
+                  "movable": "false",
+                  "deletable": "false",
+                  "fields": {
+                    "sound": "beats/groovy_beat"
+                  }
+                }
+              }
+            }
+          ]
+        }
+      },
+      "text": "####Choose a Sound\n\nThere are tons of sounds you can choose from. Change up the beat!\n\nClick the name of the sound to select a new one, and then press Run.\n",
```

Comment:
Can we use `longInstructions` here instead of `levelData.text` to get automatic loc support and the existing levelbuilder markdown editor?

Reply:
Done.